docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -39,7 +40,11 @@ contents:
 
 The config file is specified in JSON format. It contains rules that define which
 users have access to which resources. The rules are read from top to bottom and
-the first matching rule is applied. If no rule matches, access is denied.
+the first matching rule is applied. If no rule matches, access is denied. A JSON
+pointer (RFC 6901) can be specified using the ``security.json-pointer`` property
+to specify a nested object inside the JSON content containing the rules. Per default,
+the file is assumed to contain a single object defining the rules rendering
+the specification of ``security.json-pointer`` unnecessary in that case.
```

Comment:
I think it would be good to give an example how to configure it with rest endpoint and non default json pointer.